sdk/storage/storage-queue/samples-dev/queueClient.ts

```diff
@@ -62,11 +62,14 @@ export async function main() {
     `Created queue ${queueClient.name} successfully, service assigned request ID: ${createQueueResponse.requestId}`
   );
 
-  // Send a message into the queue using the sendMessage method.
-  const enqueueQueueResponse = await queueClient.sendMessage("Hello World!");
-  console.log(
-    `Sent message successfully, service assigned message ID: ${enqueueQueueResponse.messageId}, service assigned request ID: ${enqueueQueueResponse.requestId}`
-  );
+  // Send three messages into the queue using the sendMessage method.
+  const messages = ["First message", "Second Message", "Third Message"];
+  for (let i = 0; i < messages.length; i++) {
+    const enqueueQueueResponse = await queueClient.sendMessage(messages[i]);
+    console.log(
+      `Sent message successfully, service assigned message ID: ${enqueueQueueResponse.messageId}, service assigned request ID: ${enqueueQueueResponse.requestId}`
```

Comment:
I might be wrong, but I recall that ${enqueueQueueResponse.requestId}` is client request id?

Reply:
I am unsure about this too. I just took it from the previous sample without modifying it.

Reply:
@EmmaZhu There are client request id and request id. Please log a new issue if you think we need to change that.